scripts/bench.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
I think the name of the script is not explicit. It's goal is to compute the difference between two benchmarks, not to run a benchmark, hence I think the name is confusing